tests/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.1'
+
+services:
+
+  db:
+    image: postgres
```

Comment:
could you add the tag to the latest supported version of postgres?

Reply:
just saw [our docs](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/postgresql-monitoring-integration/) that say 9.0 or higher , not sure if that is a safe way state compatibility.

Reply:
Sure!